src/lib/components/table/components/TableCell/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { HTMLProps, ReactNode } from "react";
```

Comment:
This goes for all of the components:
As discussed in one of our FEJFs we should not use `index.tsx` and `style.module.scss` and instead use `ComponentName.tsx` and `ComponentName.module.scss` to make it easier to reason what you're working on. The final export from DS can stay as is, but all other internal exports and imports don't need an index file.